icons/beef.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <path d="M5.204 12.65c2.815-1.07.643-5.715 4.06-8.59 2.482-2.088 6.511-1.529 8.487 1.515 1.976 3.044 1.719 8.591-4.967 11.556-4.531 2.009-7.858 1.37-8.933-.284-1.076-1.653-.511-3.487 1.353-4.197Z"/>
+ <path d="m4.017 17.102 2.35 2.824c1.076 1.66 4.563 2.628 8.934.281 6.716-3.605 6.947-8.548 4.972-11.603l-3.256-4.002"/>
+ <path d="M10.684 8.667a2 2 0 1 1 3.986.34 2 2 0 0 1-3.986-.34Z"/>
```

Comment:
This `<path>` should in practice be a `<circle>`, also it'd be great if this icon's pixel perfection could be somewhat improved upon, it's a bit blurry on 100%. There are also a few misaligned paths and sharp corners:
![image](https://user-images.githubusercontent.com/17746067/199505601-4b1f2c39-37bc-4116-8f4d-9722298b4082.png)

As for the blurriness, I think it should be resolved if we stuck to these guides:
![image](https://user-images.githubusercontent.com/17746067/199506271-83aa84f9-d47a-4963-b29b-c74519aa766d.png)